magda-web-client/src/actions/recordActions.ts

```diff
@@ -152,6 +158,41 @@ export function fetchDistributionFromRegistry(id: string): any {
     };
 }
 
+const DefaultCkanExportProperties: CkanExportAspectProperties = {
+    status: "withdraw",
+    hasCreated: false,
+    exportRequired: false,
+    exportAttempted: false
+};
+
+async function notifyCkanExportMinion(datasetId: string) {
```

Comment:
The changes in this file were used by the turn on / off button on dataset page.
As we now control export-minion on/off from add dataset workflow, all changes in this file should not be required anymore.